.github/workflows/cdelivery-ecs-backend-caller.yml

```diff
@@ -99,3 +99,5 @@ jobs:
       meilisearch-url: ${{ secrets.MEILISEARCH_URL }}
       meilisearch-master-key: ${{ secrets.MEILISEARCH_MASTER_KEY }}
       meilisearch-rebuild-secret: ${{ secrets.MEILISEARCH_REBUILD_SECRET}}
+      openai-api-key: ${{ secrets.OPENAI_API_KEY }}
```

Comment:
Do we want an environment specific variables?

Reply:
@pyphilia It has been setup as a by environnement secret, see in GitHub environnements